pkg/operator/periodicbackupcontroller/periodicbackupcontroller.go

```diff
@@ -79,6 +79,11 @@ func (c *PeriodicBackupController) sync(ctx context.Context, _ factory.SyncConte
 	if err != nil {
 		return fmt.Errorf("PeriodicBackupController could not list backup CRDs, error was: %w", err)
 	}
+	// skip if default backup enabled
```

Comment:
@tjungblu  sooo :D

Reply:
you're skipping the whole controller. 

What I had in mind was something like:

```suggestion
	// ignore the reconciliation of the default backup
	backups.Items = slices.DeleteFunc(backups.Items, func(b backupv1alpha1.Backup) bool {
		return b.Name == "default"
	})
```

Reply:
achsooo .. but this way we could have other `Backup` CRs running alongside the default, correct ?

Reply:
sure, why not?

Reply:
yes its gr8 idea actually, updated 👍🏽